src/solo8.cpp

```diff
@@ -173,54 +130,34 @@ void Solo8::acquire_sensors()
     * Additional data
     */
   // acquire the slider positions
-  for (unsigned i=0 ; i<slider_positions_.size() ; ++i)
-  {
-    // acquire the slider
-    slider_positions_(i) = sliders_[i]->get_measurement()->newest_element();
-    // acquire the current contact states
-    contact_sensors_states_(i) =
-        contact_sensors_[i]->get_measurement()->newest_element();
-  }
+  // TODO: Acquire via Arduino.
```

Comment:
Just a suggestion: /** @todo ... */ display the todo in the documentation. It is nice when you want to track the todo-list.

Reply:
Does `// @todo` also work?